README.md

```diff
@@ -5,7 +5,7 @@ FreeClimb is a cloud-based application programming interface (API) that puts the
 This SDK is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
 
 - API version: 1.0.0
-- Package version: 2.2.0
+- Package version: 3.0.0
```

Comment:
Minor update over major

Reply:
I don't think we should make these changes a minor change in the Ruby SDK, while they are major change in the Java SDK. It could cause confusion.
https://github.com/FreeClimbAPI/java-sdk/pull/13